continuous-integration/starter-kits/README.md

```diff
@@ -1,6 +1,177 @@
+   that these are created during this configuration** 
+```
+/*-- [INSERT ID_OF_STORED_MAVEN_SETTINGS HERE] --*/
+/*-- [INSERT ID_OF_STORED_SSH_KEY HERE] --*/
+/*-- [INSERT URL_TO_GITHUB_SERVER HERE] --*/
+/*-- [INSERT URL_TO_ORG HERE] --*/
+```
+6. Commit changes to the Git repository for testing. 
+7. Create a [Jenkins Multibranch Pipeline Job](https://www.cloudbees.com/blog/jenkins-multibranch-pipeline-with-git-tutorial) 
+configured to point at the Jenkinsfile at `.ci/Jenkinsfile-buildDeploy`
+8. Test and debug using [Jenkins build logging](https://opensource.com/article/19/9/intro-building-cicd-pipelines-jenkins) 
+that reports build progress. **Note: Click the build numbeer to view build logs.**
+ 
+### Maven `multimodule` project
+Note that this differs from the `simple` project by introducing ordered, modular builds, so 
+one repo can build multiple modules. Configuration differences center around the creation of 
+`modules`, or subprojects.
+
+Starter Kit:
+- [Maven Multimodule Project Root](https://github.com/nasa-ammos/slim/blob/main/continuous-integration/starter-kits/compile-release-publish/java-maven/multimodule/)
+
+To leverage this project, make sure to do the following: (Of course, using earlier requirements.)
+1. [Create a new Git repo](https://docs.github.com/en/get-started/quickstart/create-a-repo) for 
+the multimodule job. This is an example that compiles source from one or more repositories to build 
+a single project.
+2. Check out one or more Git repositories to convert into the new project.
+3. Copy the `multimodule` project layout onto the folder from item (1).  
+4. Move source and configuration files into place per the [Maven Standard Directory Layout](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html)
+   1. Source and package structure should land in the `src/main/java` directory.
+   2. Configurations should land in the `src/main/resources` directory.
+   3. Unit tests should be moved into place similarly in the `src/test/...` directory tree.
+5. Modify the POM to support the simple project. 
+   1. **Set project-specific identification values from the following list.** These values uniquely 
+   identify the project within the Maven ecosystem by associating 
+   [Maven coordinates](https://maven.apache.org/guides/mini/guide-naming-conventions.html). [Configured 
+   URLs to servers](https://maven.apache.org/scm/maven-scm-plugin/usage.html) hosting the 
+   repository are also required. These values rely on the inheritance of a parent configuration. 
+   **Note that values** for the `PARENT_ID` and `PARENT_VERSION` coordinates apply the JPL enterprise 
+   Maven parent to the project. This will vary outside the Lab. 
+   2. Search and replace values in the example [POM file](https://github.com/nasa-ammos/slim/blob/main/continuous-integration/starter-kits/compile-release-publish/java-maven/multimodule/pom.xml) 
```

Comment:
Broken link

Reply:
Same breakage as before. ... (Needs to be merged.)